app/views/animals/show.html.erb

```diff
@@ -5,9 +5,13 @@
     <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis odio sed, aliquid magni, porro consequatur dolorum voluptatem laborum autem ipsam.</p>
     <div class="card" style="width: 24rem;">
```

Comment:
The form should really be a partial. I think it would be best to store it in views/bookings/_form.html.erb, and then render it using relative ../bookings path inside views/animals.show.html.erb